### ISSUE TITLE
Feature: Get Track Details

### ISSUE THREAD

**Issue by p5quared** [p5quared/apple-music.nvim#15]
From conversation on #13.

Need to decide what details? Guess it doesn't matter we can always add more.

2 options:
1. Individual functions (`get_track_artist()`, `get_track_album()`...).
2. `get_track_info() -> detailsTable`

I kind of like 2. more. Thoughts?

**Comment by p5quared** [comment]
I think for now getting the info of the currently playing track is plenty good enough. If someone wants we could add `get_track_details(trackname)`

**Comment by ducktordanny** [comment]
Yeah, I think the 2. option would give more flexibility to the users. And we wouldn't have to create X function to access X type of details.

**Comment by p5quared** [comment]
Exactly, we just add fields to the table. We'll just need to be clear that the table fields may be extended between releases so people shouldn't iterate over it.

**Comment by p5quared** [comment]
We could start with
1. Track Name
2. Track Artist
3. Track Album

And expand if anyone wants.
